extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergCatalogAdapter.java

```diff
@@ -132,13 +125,13 @@ static io.deephaven.qst.type.Type<?> convertPrimitiveType(final Type icebergType
     }
 
     @SuppressWarnings("unused")
-    public List<TableIdentifier> listTables(final Namespace namespace) {
+    public List<TableIdentifier> listTables(@NotNull final Namespace namespace) {
```

Comment:
Do we want to move from exposing `Namespace` and `TableIdentifier` and just use Strings?
I think Java uses the POJOs, since they can be made from Strings by the user.

Python, I *THINK*, should accept a single string (and delegate to `TableIdenfier.parse`) unioned with a sequence of strings (`TableIdentifier.of`).

We may have to do our own splitting for methods that take just a `Namespace`, since there's no `Namespace.parse`, but it's literally just dot-split.